src/NuGet.Core/NuGet.Protocol/Converters/NuGetFrameworkConverter.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Xml.Linq;
+using Newtonsoft.Json;
+using NuGet.Frameworks;
+
+namespace NuGet.Protocol.Converters
+{
+    public class NuGetFrameworkConverter : JsonConverter
+    {
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            serializer.Serialize(writer, value.ToString());
```

Comment:
nitpick: this is... complicated 😆 

The TFM value most people are used to reading, is not the value returned by `.ToString()`, but the value returned by `.GetShortFoldername()`.  And for some reason, `.nuspec` files use a 3rd format, which is what we'll probably see from package sources when deserializing dependency groups. 

But since this is a generic `NuGetFramework` converter, it's hard to say what's "correct" to use. However, I don't know of any scenario where NuGet will serialize this, so this method will probably never be used.

I wonder if we can use [this overload of `JsonConverterAttribute`](https://www.newtonsoft.com/json/help/html/M_Newtonsoft_Json_JsonConverterAttribute__ctor_1.htm) to pass data to the converter about which format type is preferred for the specific file.

Reply:
I like your comment but since this is targeted for 17.2 P3 I believe I don't have enough time for this at this time, do you want me to investigate this further for next release?

Reply:
Should we throw a not implemented exception here if there is going to be no callers for this method? We can address Andy's feedback in a follow up PR.